src/types/person.rs

```diff
@@ -21,34 +22,58 @@ pub struct Person {
 }
 
 impl Person {
-    /// Constructs a new person from a chunk vector according to the specs of
-    /// [Nicolas Markey in "Tame the BeaST"][taming], pp. 23-24.
+    /// Constructs a new person from a chunk vector:
+    /// 1. according to the specs of
+    ///     [Nicolas Markey in "Tame the BeaST"][taming], pp. 23-24.
+    /// 2. biblatex extended name format according to the
+    ///     [documentation of biblatex][biblatex], section 3.4 pp. 80-81,
+    ///     and section §4.2.3 pp. 164-165.
+    ///     Support is limited to default `nameparts`: prefix, family,
+    ///     suffix, given.
     ///
     /// [taming]: https://ftp.rrze.uni-erlangen.de/ctan/info/bibtex/tamethebeast/ttb_en.pdf
+    /// [biblatex]: https://ctan.gutenberg-asso.fr/macros/latex/contrib/biblatex/doc/biblatex.pdf
     pub fn parse(chunks: ChunksRef) -> Self {
-        let num_commas: usize =
-            chunks
-                .iter()
-                .map(|val| {
-                    if let Chunk::Normal(s) = &val.v {
-                        s.matches(',').count()
-                    } else {
-                        0
-                    }
-                })
-                .sum();
+        // Extended Name Format
+        if chunks.iter().any(|val| val.v.get().contains("=")) {
+            return Self::parse_extended_name_format(chunks);
+        }
+        // BibTex
+        Self::parse_bibtex(chunks)
+    }
+
+    /// Constructs new person from chunk slices.
+    fn parse_extended_name_format(chunks: ChunksRef) -> Self {
+        let mut person = HashMap::new();
+        for chunk in split_token_lists(chunks, ",") {
+            let (key, value) = split_at_normal_char(&chunk, '=', true);
+            let key = key.format_verbatim();
+            let value = value.format_verbatim();
+            person.insert(key, value);
+        }
+
+        let name = person.get("family").map(ToString::to_string).unwrap_or_default();
```

Comment:
I noticed that, given that you are discarding the `HashMap` afterwards, we can save a clone by taking the owned `String` from the map with
```suggestion
        let name = person.remove("family").unwrap_or_default();
```